libvast/fbs/data.fbs

```diff
@@ -30,6 +30,7 @@ table String {
 
 table Pattern {
   value: string;
+  case_insensitive: bool;
```

Comment:
> **Note**
> This was a bug fix, as case-insensitive patterns were not persisted correctly. This change is backwards-compatible.

Reply:
Maybe this bug fix can be extracted into a separate `re2`-less pull request, just for the 3.0 release.

Reply:
For this, and also the others below: Feel free to just create a PR with just the first commit of this branch if you don't want to wait for me to be back from vacation next week. The CI runs through for that, it just doesn't for the second commit.